<launch>
<node pkg="turtlesim" type="turtlesim_node" name="turtle1" output="screen" />
<node pkg="turtlesim" type="turtle_teleop_key" name="key_control" output="screen" />

<node pkg="learning_tf" type="create_turtle" name="turtle2" output="screen" />

<node pkg="learning_tf" type="turtle_tf2_pub" name="caster1" output="screen" args="turtle1" />
<node pkg="learning_tf" type="turtle_tf2_pub" name="caster2" output="screen" args="turtle2" />

<node pkg="learning_tf" type="turtle_tf2_sub" name="listener" output="screen" />

</launch>